src/awake.hpp

```diff
@@ -739,7 +738,8 @@ enum {
 #define META_QUICKENING    6
 #define META_REFLECTING    7
 #define META_SHIELDING    8
-#define META_MAX    9
+#define META_ANCHORING    9
+#define META_MAX    10
```

Comment:
Whenever you change something in this file, you also usually need to change constants.cpp. To do that, search that for 'anchoring' and update the array accordingly, leaving some placeholder string in the current spot where anchoring is so the ordering doesn't get messed up.

Reply:
Got it! Thanks for the help.